kitten-app/src/stateHooks/useForm.ts

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import createCatServices from "../services/createCatServices";
+
+// useForm functional component
```

Comment:
[question] Is this a functional component or is this called something different?